fairseq/models/roberta/model.py

```diff
@@ -230,15 +231,18 @@ def forward(self, features, masked_tokens=None, **kwargs):
 class RobertaClassificationHead(nn.Module):
     """Head for sentence-level classification tasks."""
 
-    def __init__(self, input_dim, inner_dim, num_classes, activation_fn, pooler_dropout):
+    def __init__(self, input_dim, inner_dim, num_classes, activation_fn, pooler_dropout, sequence_tagging):
         super().__init__()
         self.dense = nn.Linear(input_dim, inner_dim)
         self.activation_fn = utils.get_activation_fn(activation_fn)
         self.dropout = nn.Dropout(p=pooler_dropout)
         self.out_proj = nn.Linear(inner_dim, num_classes)
+        self.sequence_tagging = sequence_tagging
 
     def forward(self, features, **kwargs):
-        x = features[:, 0, :]  # take <s> token (equiv. to [CLS])
+        x = features
+        if not self.sequence_tagging:
+            x = x[:, 0, :]  # take <s> token (equiv. to [CLS])
         x = self.dropout(x)
         x = self.dense(x)
         x = self.activation_fn(x)
```

Comment:
@myleott Would it make sense to add `layer_norm` here just like in the `RobertaLMHead`? https://github.com/pytorch/fairseq/pull/1709/files#diff-ad4f875b91a61dcc1511c93c8e38cc4bL224